src/election.jl

```diff
@@ -41,14 +41,37 @@ function count_votes(name::String, election::Election)::DistrictScore
         vote_totals = sum(election.vote_counts[district, :])
         vote_shares = election.vote_counts[district, :] ./ vote_totals
         election.vote_shares[district, :] = vote_shares
```

Comment:
Updating the `Election` object (now in the new code, this is only performed by `vote_updater`)